### PR TITLE
Adds a few loadout items

### PR DESCRIPTION
**Pull request by Krypandenej**
## About The Pull Request
Adds the xeno action figure, a mime-only lighter, and some Leary's Delight cigs (for RDS havers) to the loadout.

## Why It's Good For The Game
More variety is always good, and these shouldn't break the balance of the game. We already have mindbreaker pills, too.

## Proof Of Testing
image
image

## Changelog

:cl:
add: Added Leary's Delight cigs, a xeno action figure, and a pale zippo to loadout
/:cl:

### CODE REVIEW

**Comment by Krypandenej** [comment]
2nd PR woooo yaaaaaay lemme know if I need to do something to any of this, I'm p sure the weird text below the mime lighter is a UI glitch

File: modular_zubbers/code/modules/loadout/categories/pocket.dm [hunk 1]
```diff
@@ -70,6 +70,15 @@
 	name = "Pocket Watch"
 	item_path = /obj/item/clothing/accessory/pocketwatch
 
+/datum/loadout_item/pocket_items/mimezippo
+	name = "Pale Zippo"
+	item_path = /obj/item/lighter/royal
+	restricted_roles = list(JOB_MIME)
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/loadout/categories/pocket.dm:76]: Already available as the "Royal Zippo" lighter

```suggestion
```

**Review by Majkl-J — CHANGES_REQUESTED** [review]

**Review by Majkl-J — APPROVED** [review]
Nice

File: modular_zubbers/code/modules/loadout/categories/pocket.dm (hunk 1 above)
Comment (Krypandenej) [reply modular_zubbers/code/modules/loadout/categories/pocket.dm]: WOOPS wrong filepath

**Pull request merged**